packages/repository/test/unit/repositories/has-one-repository-factory.unit.ts

```diff
@@ -142,18 +142,23 @@ describe('createHasOneRepositoryFactory', () => {
         type: 'string',
         required: true,
       })
+      .addProperty('id', {
+        type: 'number',
+        id: true,
```

Comment:
I am little bit confused. The model name is `ModelWithoutIndexFK`, but it has a regular `id` property. What am I missing?

Reply:
Good question. I added an `id` property because the model itself extends from `Entity`, but the purpose here is the decorated relational property is not part of the composite index, thus the name `ModelWithoutIndexFK`. Thoughts on improving it if it's confusing? I can extend from `Model` and remove the `id` property as well.

Reply:
>  I can extend from Model and remove the id property as well.

If this works, then I think it would be a good solution.

If not, then please capture the information from your comment above in a code comment.